src/include/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.15)
+
+#####################################################
+# Kompute
+#####################################################
+
+target_include_directories(kompute PUBLIC $<INSTALL_INTERFACE:include>
+                                          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+
+target_sources(kompute PRIVATE
+     # Header files (useful in IDEs)    
+    kompute/Algorithm.hpp
+    kompute/Core.hpp
+    kompute/Kompute.hpp
+    kompute/Manager.hpp
+    kompute/Sequence.hpp
+    kompute/Tensor.hpp
+
+    kompute/operations/OpAlgoDispatch.hpp
+    kompute/operations/OpBase.hpp
+    kompute/operations/OpMemoryBarrier.hpp
+    kompute/operations/OpMult.hpp
+    kompute/operations/OpTensorCopy.hpp
+    kompute/operations/OpTensorSyncDevice.hpp
+    kompute/operations/OpTensorSyncLocal.hpp
+
+    kompute/shaders/shaderlogisticregression.hpp
+    kompute/shaders/shaderopmult.hpp
+
+    kompute/logger/Logger.hpp
+)
+
+install(DIRECTORY kompute DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+#####################################################
+# Logger
+#####################################################
+
+target_include_directories(kp_logger PUBLIC $<INSTALL_INTERFACE:include>
+                                         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+
+target_sources(kp_logger PRIVATE
```

Comment:
What is the main reason to separate the logger into a different cmake package instead of just a header file within the core kompute?

Reply:
To reduce compile time and to keep everything logically separated.